stl/inc/system_error

```diff
@@ -67,9 +67,13 @@ _EXPORT_STD _NODISCARD const error_category& system_category() noexcept;
 
 _EXPORT_STD class __declspec(novtable) error_category { // categorize an error
 public:
+#ifndef _M_CEE_PURE

```

Comment:
No change requested (just want to mention this): As a general rule, we prefer to avoid negation in `if` statements, preprocessor conditions, etc. when doing so doesn't introduce any additional complexity. However, this is a relatively weak preference, and if there's any kind of reason to prefer a negated test, it's fine to keep. Here, "not pure" is the normal case, so putting it first is reasonable, relegating the weird `/clr:pure` fallback to the bottom, so that's good.